src/main/java/org/breskul/connectivity/annotation/Entity.java

```diff
@@ -0,0 +1,11 @@
+package org.breskul.connectivity.annotation;
```

Comment:
What is the value of the annotation Entity?  Does it make sense if we have a Table already?

Reply:
This is how hibernate works

Reply:
We don't have-to copy-paste Hibernate. And it was advised to have the most straightforward configuration. That's why I asked if this annotation is necessary for us.